### ISSUE TITLE
Flickr30k image text retrieval CustomTrainDataset

### ISSUE THREAD

**Issue by guanhdrmq** [NielsRogge/Transformers-Tutorials#277]
Hi team,

How are you? Could you share a demo customer data loader for flickr30k? Because this task has no label and not sure how to write the " encoding["label"] = torch.tensor(label)" part.

Thank you very much.
